src/types/geography.ts

```diff
@@ -0,0 +1,301 @@
+   */
+  get latitude(): number {
+    return this.point?.latitude || 0;
+  }
+
+  /**
+   * Return Spatial Reference System Identifier (SRID), identifies de reference system for the represented Geographic object
+   * @returns number
+   */
+  get srid(): number {
+    // TODO: not implemented
+    return 0;
+  }
+
+  /**
+   * Return true if the Geography is emtpy
+   * @returns boolean
+   */
+  isEmpty(): boolean {
+    return this.featureType === "";
+  }
+
+  /**
+   * Return true if the Geography is null
+   * @returns boolean
+   */
+  isNull(): boolean {
+    return this.featureType === "";
+  }
+
+  /**
+   * Set Geography to empty value
+   */
+  setEmpty() {
+    this.point = undefined;
+    this.line = undefined;
+    this.polygon = undefined;
+    this.featureType = "";
+  }
+
+  /**
+   * Serializes the object to string (alias for toWkt)
+   * @returns string
+   */
+  toString(): string {
+    return this.toWkt();
+  }
+
+  /**
+   * Serializes the object to WKT representation
+   * @returns string
+   */
+  toWkt(): string {
+    switch (this.featureType) {
+      case "POINT":
+        return `POINT (${this.longitude} ${this.latitude})`;
+      case "LINE":
+        return `LINESTRING (${this.line
+          .map(point => `${point.longitude} ${point.latitude}`)
+          .join(", ")})`;
+      case "POLYGON":
+        return `POLYGON (${this.polygon
+          .map(
+            ring =>
+              `(${ring
+                .map(point => `${point.longitude} ${point.latitude}`)
+                .join(", ")})`
+          )
+          .join(", ")})`;
+      default:
+        return "";
+    }
+  }
+
+  /**
+   * Serializes the object to GeoJSON representation
+   * @returns string
+   */
+  toGeoJson(): string {
+    switch (this.featureType) {
+      case "POINT":
+        return `{"type":"Point","coordinates":[${this.longitude},${this.latitude}]}`;
+      case "LINE":
+        return `{"type":"LineString","coordinates":[${this.line
+          .map(point => `[${point.longitude},${point.latitude}]`)
+          .join(",")}]}`;
+      case "POLYGON":
+        return `{"type":"Polygon","coordinates":[${this.polygon
+          .map(
+            ring =>
+              `[${ring
+                .map(point => `${point.longitude} ${point.latitude}`)
+                .join(",")}]`
+          )
+          .join(",")}]}`;
+      default:
+        return '{"type":"GeometryCollection","coordinates":[]}';
+    }
+  }
+
+  /**
+   * Return new instance of this Geography object
+   * @returns Geography
+   */
+  toGeoPoint(): Geography {
+    return new Geography(this.toString());
+  }
+
+  /**
+   * Return new instance of this Geography object
+   * @returns Geography
+   */
+  toGeoLine(): Geography {
+    return new Geography(this.toString());
+  }
+
+  /**
+   * Return new instance of this Geography object
+   * @returns Geography
+   */
+  toGeoPolygon(): Geography {
+    return new Geography(this.toString());
+  }
+
```

Comment:
Maybe we can centralize the code for cloning in a private method?

```suggestion
  /**
   * Return new instance of this Geography point object
   * @returns Geography
   */
  toGeoPoint(): Geography {
    return this.clone();
  }

  /**
   * Return new instance of this Geography line object
   * @returns Geography
   */
  toGeoLine(): Geography {
    return this.clone();
  }

  /**
   * Return new instance of this Geography polygon object
   * @returns Geography
   */
  toGeoPolygon(): Geography {
    return this.clone();
  }
  
  private clone(): Geography {
    return new Geography(this.toString());
  }
```